GraphQL-TSC.md

```diff
@@ -207,6 +207,12 @@ The TSC is responsible for defining a policy and a naming convention for any rep
 
 Each repository will be managed by the EasyCLA signature tool, ensuring that all contributors have agreed to the GraphQL Specification Membership Agreement.
 
+### Speaking on behalf of the GraphQL Project
+
+GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). However when speaking on behalf of the GraphQL project (e.g. technical docs, project websites, READMEs, or anything merged into a git repository under the GraphQL organization) we do not allow posting opinions or messages of political advocacy using the voice of the project without prior approval via GraphQL TSC vote.
```

Comment:
I would guess the requirement to adhere to the Code of Conduct is meant only for advocacy included within Github issues and PR discussions. I would make this explicit, if this is indeed the intent, by ending the sentence "(eg. Twitter, blog)." and beginning a new sentence "Individuals are also welcome to do so within Github..."

Reply:
CoC actually can apply outside project spaces in some cases - https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md#scope